packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/utils/heatmap.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../types.dart';
+import 'maps_object.dart';
+
+/// Converts an [Iterable] of Heatmaps in a Map of
+/// HeatmapId -> Heatmap.
+Map<HeatmapId, Heatmap> keyByHeatmapId(
+  Iterable<Heatmap> heatmaps,
+) {
+  return keyByMapsObjectId<Heatmap>(heatmaps).cast<HeatmapId, Heatmap>();
+}
+
+/// Converts a Set of Heatmaps into something serializable in JSON.
+Object serializeHeatmapSet(Set<Heatmap> heatmaps) {
```

Comment:
We should be able to remove this as well.

Reply:
This is used by the platform interface for some reason. Not sure if it's safe to remove without other refactoring.